app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/ContractDetailViewModel.kt

```diff
@@ -5,23 +5,27 @@ import androidx.lifecycle.viewModelScope
 import arrow.core.raise.either
 import com.hedvig.android.core.common.RetryChannel
 import com.hedvig.android.core.demomode.Provider
-import com.hedvig.android.feature.insurances.insurancedetail.data.ContractDetails
-import com.hedvig.android.feature.insurances.insurancedetail.data.GetContractDetailsUseCase
+import com.hedvig.android.feature.insurances.data.GetInsuranceContractsUseCase
+import com.hedvig.android.feature.insurances.data.InsuranceContract
+import kotlin.time.Duration.Companion.seconds
 import kotlinx.coroutines.flow.SharingStarted
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.WhileSubscribed
 import kotlinx.coroutines.flow.stateIn
-import kotlin.time.Duration.Companion.seconds
 
 internal class ContractDetailViewModel(
   contractId: String,
-  private val getContractDetailsUseCase: Provider<GetContractDetailsUseCase>,
+  private val getInsuranceContractsUseCase: Provider<GetInsuranceContractsUseCase>,
```

Comment:
```suggestion
  private val getInsuranceContractsUseCaseProvider: Provider<GetInsuranceContractsUseCase>,
```

Maybe? We do follow that convention usually with this, so that doing `provide()` on it does not look odd further down. What do you think?